release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.InterAppCalls/Skyline_DataMiner_Core_InterAppCalls_Range_1.1.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: Skyline_DataMiner_Core_InterAppCalls_Range_1.1
+---
+
+# Skyline DataMiner Core InterAppCalls Range 1.1
+
+> [!NOTE]
+> Range 1.0.x.x is supported as from **DataMiner 10.4.0**.
```

Comment:
Is there also a feature release version we can add here, for people on the feature release track?